stable_plugins/classical_ml/data_preparation/pandas_preprocessing/templates/pd_preprocessing_step_2.html

```diff
@@ -2,12 +2,30 @@
 
 {% block head %}
     {{ super() }}
+    <style>
+        ul.qhana-form-input {
+            max-height: 150px;
```

Comment:
I am not sure about the max height and hidden overflow. This will just clip long entries, which is bad for usability. It is usually better to have a UI that looks bad but at least works (and in this case: shows everything) in such edge cases instead of one that always looks great but is unusable in some cases.

Reply:
Is "overflow-wrap: break-word;" also a viable option?
It inserts line breaks, if the label of a checkbox is too long. Though, this does not ensure that the whole word is visible at once, but maybe this is a good compromise between usability and looks. What do you think?

Reply:
Allowing breaks inside a word is better, but only if the height is not limited (or scrolls on overflow). Otherwise, part of the label will still be hidden in the overflow.

Reply:
With overflow auto in one direction it should be ok (I have missed the `auto` after `hidden` in the css)